contents/dataframes_select.md

```diff
@@ -24,29 +24,29 @@ s = """select(responses(), "id", "q1", "q2")"""
 sco(s, process=without_caption_label)
 ```
 
-To select everything _except_ one or more columns, use `Not` with either a single column name:
+To select everything _except_ one or more columns, use `Not` with either a single column:
 
 ```jl
 s = """select(responses(), Not(:q5))"""
 sco(s, process=without_caption_label)
 ```
 
-Or, with multiple column names:
+Or, with multiple columns:
 
 ```jl
 s = """select(responses(), Not([:q4, :q5]))"""
 sco(s, process=without_caption_label)
 ```
 
-It's also fine to combine column names with `Not`:
+It's also fine to mixmatch columns that we want to preserve with columns that we do `Not` want to select:
```

Comment:
```suggestion
It's also fine to mix and match columns that we want to preserve with columns that we do `Not` want to select:
```
https://en.wiktionary.org/wiki/mixmatch

Reply:
ok openned an issue #70 that covers both mix and match and use case